packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,17 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map or as these directories:\n  ` +
+                searchQueue.join(`\n  `)
```

Comment:
now that node-haste is back in the tree, I will make the name of the cache directory recognizable again – atm it’s only a hash, which isn’t exactly convenient.